File: src/Workflow.Web.Service/Workflow/app/model/itRequestForm/RequestUser.js
Ext.define('Workflow.model.itRequestForm.RequestUser', {
    extend: 'Workflow.model.Base',
	
    fields: [
        {name: 'id',     		type: 'int'		},
        {name: 'empId',         type: 'int'	},
        {name: 'empNo',     	type: 'string'	},
		{name: 'empName',     	type: 'string'	},
		{name: 'teamId',     	type: 'int'	    },
		{name: 'teamName',     	type: 'string'	},
		{name: 'position',     	type: 'string'	},
		{name: 'email',     	type: 'string'	},
        {name: 'hiredDate',     	type: 'date'	},
        {name: 'manager',     	type: 'string'	},
        {name: 'phone',     	type: 'string'	}
    ]
});

File: src/Workflow.Domain/Entities/RequestUser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Workflow.Domain.Entities.Base;

namespace Workflow.Domain.Entities
{
    public class RequestUser
    {
        public int Id { get; set; }
        public int RequestHeaderId { get; set; }
        public virtual RequestHeader RequestHeader { get; set; }

        public string EmpId { get; set; }
        public string EmpName { get; set; }
        public string EmpNo { get; set; }
        public int TeamId { get; set; }
        public virtual Department Team { get; set; }

        public string Position { get; set; }
        public string Email { get; set; }
        public DateTime? HiredDate { get; set; }
        public string Manager { get; set; }
        public string Phone { get; set; }
        public Nullable<int> Version { get; set; }
    }   
}